core/src/main/java/io/questdb/griffin/engine/functions/bool/InLongFunctionFactory.java

```diff
@@ -161,12 +174,12 @@ private static long parseValue(IntList argPositions, Function func, int i) throw
     }
 
     private static class InLongConstFunction extends NegatableBooleanFunction implements UnaryFunction {
-        private final LongList inList;
+        private final LongHashSet inList;
```

Comment:
Nit: rename `inList` to `inSet` (here and below). I got confused at first and thought we don't use a hash set.